alpha/apps/kaltura/lib/kPermissionManager.php

```diff
@@ -1093,6 +1098,64 @@ public function objectCreated(BaseObject $object)
 		return true;
 	}
 	
+	protected static function handleRecycleBinPermission($permission)
```

Comment:
worth considering storing the max days on this profile as well and not on partner level